website/docs/r/host.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "zabbix"
+page_title: "Zabbix: zabbix_host"
+sidebar_current: "docs-zabbix-resource-host"
+description: |-
+  Provides a zabbix host resource. This can be used to create and manage Zabbix Host.
+---
+
+# zabbix_host
+
+An [host](https://www.zabbix.com/documentation/current/manual/api/reference/host) is typically a device to monitor.
+
+## Example Usage
+
+Create a new host
+
+```hcl
+resource "zabbix_host_group" "demo_group" {
+  name = "Template demo group"
+}
+
+resource "zabbix_host" "demo_host" {
+  host = "127.0.0.1"
+  name = "the best name"
+  interfaces {
+    ip = "127.0.0.1"
+    main = true
+  }
+  groups = ["Linux servers", "${zabbix_host_group.demo_group.name}"]
+  templates = ["Template ICMP Ping"]
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `host` - (Required) Technical name of the host.
+* `name` - (Required) Visible name of the host.
+* `monitored` - (Optional) Whether the host is monitored or not. Can be `true` (default, monitored), `false` (not monitored).
+* `interfaces` - (Required, Multiple, Min: 1)  List of the host interfaces. Note that any changes to interface will trigger recreate.
+  * `main` - (Required) Define if it is the default interface or not. Can be `true` (default, is default interface), `false` (not default interface).
+  * `dns` - (Optional) Interface DNS name.
+  * `ip` - (Optional) Interface IP address
+  * `port` - (Optional) TCP/UDP port number of agent. Default is `10050`.
+  * `type` - (Optional) Interface type. Can be `agent` (default), `snmp`, `ipmi`, `jmx`.
+* `groups` - (Optional) List of host group names the host belongs to.
+* `templates` - (Optional) List of template names link to the host.
```

Comment:
I still think templates names are used here, not templates links

Reply:
Yes, but it's link in the way template names to link to the host. I will change in that way.

Reply:
oh yes you are right so it could be correct sorry